frontend/tests/features/collection/collection.test.ts

```diff
@@ -38,7 +38,7 @@ type CollectionFormInput = Pick<
 >;
 
 describe("Collection", () => {
-  describeIfDeployed("Logged In Tests", () => {
+  describe("Logged In Tests", () => {
```

Comment:
all the tests nested in this test block can be run in all environments, since we're not publishing a collection

Reply:
Won't this lead to failures in `e2e-test` in `push-tests.yml`?

Reply:
What kind of failures? The push tests seem to pass for this PR already. Maybe I'm missing something 😆

Reply:
Push tests are run on a local FE instance with a deployed backend, wouldn't the test runner hit the deployed Auth0 and then get redirected to the deployed FE?

Reply:
ah sorry my bad 🤦 We're only running the tests in dev, staging, and prod as shown in line 42. Just updated the original comment!